deployment/group_vars/wetc.yml

```diff
@@ -45,7 +47,9 @@ UI_FOREIGN_GAS_PRICE_FALLBACK: 10000000000
 MONITOR_PORT: 3003
 MONITOR_HOME_DEPLOYMENT_BLOCK: 7703292
 MONITOR_FOREIGN_DEPLOYMENT_BLOCK: 7412459
-MONITOR_GAS_LIMIT: 300000
-MONITOR_GAS_PRICE_FALLBACK: 21
+MONITOR_HOME_GAS_LIMIT: 300000
+MONITOR_FOREIGN_GAS_LIMIT: 300000
+MONITOR_HOME_GAS_PRICE_FALLBACK: 21
```

Comment:
Should they be equal `HOME_GAS_PRICE_FALLBACK` and `FOREIGN_GAS_PRICE_FALLBACK`?